backend/mora/service/manager.py

```diff
@@ -13,20 +13,145 @@
+        pass
+
+    @classmethod
+    def get(cls, c, type, objid):
+
+        search = {
+            SEARCH_FIELDS[type]: objid
+        }
+
+        function_effects = [
+            cls.get_one_mo_object(effect, start, end, funcid)
+            for funcid, funcobj in c.organisationfunktion.get_all(
+                funktionsnavn=cls.function_key,
+                **search,
+            )
+            for start, end, effect in c.organisationfunktion.get_effects(
+                funcobj,
+                {
+                    'relationer': (
+                        'opgaver',
+                        'adresser',
+                        'organisatoriskfunktionstype',
+                        'tilknyttedeenheder',
+                        'tilknyttedebrugere',
+                        'tilknyttedefunktioner',
+                    ),
+                    'tilstande': (
+                        'organisationfunktiongyldighed',
+                    ),
+                },
+                {
+                    'attributter': (
+                        'organisationfunktionegenskaber',
+                    ),
+                    'relationer': (
+                        'tilhoerer',
+                        'tilknyttedeorganisationer',
+                        'tilknyttedeitsystemer',
+                    ),
+                },
+            )
+            if util.is_reg_valid(effect)
+        ]
+        return flask.jsonify(function_effects)
+
+    @classmethod
+    def get_one_mo_object(cls, effect, start, end, funcid):
+        c = common.get_connector()
+
+        persons = list(mapping.USER_FIELD.get_uuids(effect))
+        manager_types = list(mapping.ORG_FUNK_TYPE_FIELD.get_uuids(effect))
+        manager_levels = list(mapping.MANAGER_LEVEL_FIELD.get_uuids(effect))
+        addresses = list(mapping.FUNCTION_ADDRESS_FIELD.get_uuids(effect))
+        responsibilities = list(mapping.RESPONSIBILITY_FIELD.get_uuids(effect))
+        org_units = list(mapping.ASSOCIATED_ORG_UNIT_FIELD.get_uuids(effect))
+
+        func = {
+            mapping.UUID: funcid,
+            mapping.RESPONSIBILITY: [
+                facet.get_one_class(c, classid, classobj)
+                for classid, classobj in c.klasse.get_all(
+                    uuid=responsibilities
+                )
+            ],
+            mapping.ORG_UNIT: orgunit.get_one_orgunit(
+                c, org_units[0],
+                details=orgunit.UnitDetails.MINIMAL),
+            mapping.VALIDITY: {
+                mapping.FROM: util.to_iso_date(start),
+                mapping.TO: util.to_iso_date(
+                    end, is_end=True)
+            },
+        }
+
+        func[mapping.ADDRESS] = []
+        for uuid in addresses:
+            orgfunc = c.organisationfunktion.get(uuid=uuid)
+            try:
+                addr = address.get_one_address(orgfunc)
+            except IndexError as e:
+                func[mapping.ADDRESS].append(None)
```

Comment:
This looks a bit iffy? Could you explain why it's needed? Adding `None` seems a bit odd compared to just skipping the value. Also; what would raise `IndexError`?

Reply:
Det er iffy. Testen forventer øjensynligt at der bliver sat None inf for addresser, der ikke findes endnu.